.github/workflows/test-integration-template.yaml

```diff
@@ -187,13 +194,14 @@ jobs:
       run: |
         echo $TEST_NAMESPACE
         kubectl delete ns --ignore-not-found=true \
-          -l "github-id=${{ steps.vars.outputs.identifier }},test-flow=${{ matrix.scenario.flow }},test-persistent=true"
+          -l "github-id=${{ steps.vars.outputs.identifier }},test-flow=${{ matrix.scenario.flow }}"
         kubectl create ns $TEST_NAMESPACE
         kubectl label ns $TEST_NAMESPACE github-run-id=$GITHUB_WORKFLOW_RUN_ID
         kubectl label ns $TEST_NAMESPACE github-job-id=$GITHUB_WORKFLOW_JOB_ID
         kubectl label ns $TEST_NAMESPACE github-id=${{ steps.vars.outputs.identifier }}
         kubectl label ns $TEST_NAMESPACE test-flow=${{ matrix.scenario.flow }}
-        kubectl label ns $TEST_NAMESPACE test-persistent=${{ env.CI_PERSISTENT }}
+        kubectl label ns $TEST_NAMESPACE repo=${{ env.CI_REPO }}
```

Comment:
The `repo` shouldn't be an input but should be set automatically from the [GH context](https://docs.github.com/en/actions/learn-github-actions/contexts).

Reply:
I get the following error in the create namespace step when automatically grabbing the repo name:
```
error: invalid label value: "repo=camunda/camunda-platform-helm": a valid label must be an empty string or consist of alphanumeric characters, '-', '_' or '.', and must start and end with an alphanumeric character (e.g. 'MyValue',  or 'my_value',  or '12345', regex used for validation is '(([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9])?')
```
I will try to make a workaround.

Reply:
we can make it simpler by having 2 labels, one for the GH org and another for the repo.

Reply:
@aabouzaid I finally got a working solution. I didn't use two variables but I used a builtin function called `basename`:
`  CI_REPO: $(basename ${{ github.event.repository.full_name }})`
It looks clean. It just grabs the repo name and not the org name. This is the resulting manifest on the namespace:
```
apiVersion: v1
kind: Namespace
metadata:
  annotations:
    cleaner/ttl: ""
  creationTimestamp: "2024-06-13T11:28:35Z"
  labels:
    github-id: gke-1984
    github-job-id: yzvnj6
    github-run-id: "9498795533"
    kubernetes.io/metadata.name: camunda-pr-1984-run-9498795533-sfx-yzvnj6
    repo: camunda-platform-helm
    test-flow: install
  name: camunda-pr-1984-run-9498795533-sfx-yzvnj6
  resourceVersion: "451177427"
  uid: 4b682356-a6bb-43af-a646-3d3bed774642
spec:
  finalizers:
  - kubernetes
status:
  phase: Active
```

What do you think? :)

Reply:
I'd say let's also have the org so it's easier to filter the namespaces per orgs.

also I'd use the default env vars since they are exposed by GHA directly in the shell.

https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables